### PR TITLE
Various TV-related fixes

### PR DESCRIPTION
**Pull request by MrTimscampi**
**Changes**

* **Make top bar non-floating on TV**
  There is really no use to having a floating top bar on TV. It can't be focused with a remote and takes up valuable space on an already very limited screen real estate
* **Fix inline CSS style for itemDetails**
  Not really a bug fix, but this removes some linting errors on an HTML file. Ideally we'd move inline CSS out, but we still need major CSS cleanup to happen before that is really a thing we can do.
* **Fix font sizes**
  The baseline was something like 14.88px instead of 16px. Now it's 16px and the TV baseline has been adjusted to be 20px, to be in line with WebOS and Tizen guideline (with comment to explain)
* **Fix scollers cutting off images when focused on TV**
  Scrollers were cutting some parts of focused cards due to the overflow being hidden. This adjusts the top/bottom margins and padding to prevent those cuts
* **Fix icon position on cards in TV layout**
  Placeholder icons were weirdly positioned. This makes them centered again in the TV layout.
* **Don't constrain end position for scrollers if in TV layout**
  When getting to the end of an emby-scroller, the focus keeps going to the right while the transform is stopped to the last item once it reaches the right of the screen. This keeps the focus centered until the end of the scroller in the TV layout (I plan to eventually rework how that focus works, since center focus is pretty terrible, but that's for another time. This at least puts a bandaid on one of the issues with them).
* **Use emblem as logo in TV layout**
  In the interest of saving previous screen real estate, makes the logo in the top bar use the emblem instead of the banner.
* **Hide admin menus if in TV layout**
  They're pretty much broken on TV, so hide them. You also shouldn't be using a TV to administer your server, so this simplifies the app on TV a bit.
* **Rework the details page for TV**
  In order to make the page work better with the focus manager, this reworks the design of the page to make it simpler to navigate using the arrow keys.  
  It also hides some stuff, like the external links, since we don't want to allow people to navigate outside of our environment on TVs, where guideline may be stricter for this kind of stuff.  
  I used a FAB for the play button (Set dynamically between play and resume, depending on what should be the primary action) in order to make it more obvious as to what the default action is.

image

**Issues**

### CODE REVIEW

File: src/themes/appletv/theme.css [hunk 1]
```diff
@@ -23,21 +23,25 @@ html {
...
+.layout-tv .skinHeader.semiTransparent {
+    background: none;
+}
+
 .pageTitleWithDefaultLogo {
     background-image: url(../../assets/img/banner-dark.png);
 }
 
+.layout-tv .pageTitleWithDefaultLogo {
+    background-image: url(../../assets/img/icon-transparent.png);
```
Comment (Camc314) [inline src/themes/appletv/theme.css:42]:
```suggestion
    background-image: url(../../assets/img/banner-light.png);
```

This is up for debate. I think it looks better with the Jellyfin text next to the logo.

Comment (MrTimscampi) [reply src/themes/appletv/theme.css:42]: This is meant to save horizontal space in the TV layout (for the tabs, mostly), as it's very limited due to everything being larger.

Ideally, we'll move the tab bar around to use the freed space a bit, but that requires other stuff to be merged in (the breakpoint PR, mostly)

Comment (Camc314) [reply src/themes/appletv/theme.css:42]: Ok, that sounds good.

**Review by Camc314 — APPROVED** [review]

File: src/controllers/itemDetails/index.js [hunk 2]
```diff
@@ -572,9 +572,13 @@ import 'emby-select';
 
         // Start rendering the artwork first
         renderImage(page, item);
-        renderLogo(page, item, apiClient);
+
+        // Same some screen real estate in TV mode
```
Comment (dkanada) [inline src/controllers/itemDetails/index.js:576]:
```suggestion
        // Save some screen real estate in TV mode
```

File: src/themes/appletv/theme.css [hunk 3]
```diff
@@ -23,21 +23,25 @@ html {
...
     color: rgba(0, 0, 0, 0.7);
     background: #303030;
-    background: -webkit-gradient(linear, left top, right top, from(#bcbcbc), color-stop(#a7b4b7), color-stop(#beb5a5), color-stop(#adbec2), to(#b9c7cb));
-    background: -webkit-linear-gradient(left, #bcbcbc, #a7b4b7, #beb5a5, #adbec2, #b9c7cb);
-    background: -o-linear-gradient(left, #bcbcbc, #a7b4b7, #beb5a5, #adbec2, #b9c7cb);
     background: linear-gradient(to right, #bcbcbc, #a7b4b7, #beb5a5, #adbec2, #b9c7cb);
 }
 
 .skinHeader.semiTransparent {
-    -webkit-backdrop-filter: none !important;
```
Comment (dkanada) [inline src/themes/appletv/theme.css:33]: Does this mean we can remove the browser prefixes now?

**Review by dkanada — APPROVED** [review]

File: src/themes/appletv/theme.css (hunk 3 above)
Comment (MrTimscampi) [reply src/themes/appletv/theme.css:33]: We could since we moved to Gulp back in pre-10.6 :) Just no one has done it x)

**Review by dkanada — APPROVED** [review]

**Pull request merged**